changelogs/unreleased/default_ruby.md

```diff
@@ -0,0 +1,22 @@
+## Default Ruby version for new apps is now 3.3.8
+
+The [default Ruby version for new Ruby applications is 3.3.8](https://devcenter.heroku.com/articles/ruby-support#default-ruby-version-for-new-apps). You’ll only get the default if the application does not specify a ruby version.
+
+Heroku highly recommends specifying your desired Ruby version. You can specify a Ruby version in your `Gemfile:
```

Comment:
Blerg, missing backtick to close out `Gemfile` here. Caught before publishing the changelog, though.